### PR TITLE
Newpunc

### PR DESCRIPTION
**Pull request by DarusoPL**
new count punctures function

### CODE REVIEW

**Comment by Veroniush** [comment]
Where is your Issue for this Request?

**Comment by lucynao** [comment]
There were many changes in master con you make sure you have the latest vertion of master before you merge code

File: Program.cs [hunk 1]
```diff
@@ -118,7 +118,9 @@ static void Main(string[] args)
                     if (File.Exists(LocalFilePath))
                     {
                         string s = File.ReadAllText(LocalFilePath);
-                        Console.WriteLine("Numer of punctuation marks: {0}", StringHelper.CountPunctuationMark(s));
+
+                        string vs = s.Replace("<", "").Replace(",", "").Replace(":", "").Replace(";", "").Replace("'", "").Replace("!","").Replace(">","");
```
Comment (lucynao) [inline Program.cs:122]: sugestion to move that to StringHelper class

**Review by lucynao — APPROVED** [review]
please check comments in code

**Comment by Veroniush** [comment]
Cmon mate why u writting code on 2 weeks ago version? Wake up!

**Pull request merged**